lib/report/general_statistics.rb

```diff
@@ -12,12 +12,77 @@ def initialize(start_date:, end_date:)
     end
 
     def report
-      {
+      @report ||= {
         all: full_report(all_users, all_logins),
         interval: full_report(interval_users, interval_logins),
       }
     end
 
+    def humanize
+      output = ""
+
+      output += "All registrations to #{@end_date}: \n#{report[:all][:users][:count]}\n\n"
+      output += "New registrations between #{@start_date} and #{@end_date}: \n#{report[:interval][:users][:count]}\n\n"
```

Comment:
Whilst we are here, I was wondering if we can make this report easier to read in genreal.

The `between #{@start_date} and #{@end_date}` always comes out the same and massively increases the line length given it's a long date string with timezone and all..

I was wondering if we could move this to the top, something like:

`Report for period between #{@start_date} and #{@end_date}`

Then turn lines like this into more simply:
`New registrations:`

What'd you think?

Reply:
If you agree but don't want this to be blocking, also happy for this to be a separate PR too

Reply:
I've been meaning to ask a content designer to have a look and the whole report. Also adding some ASCII charts would be good

Reply:
Yeah it's pretty horrible.  I'll reduce the timestamp duplication a bit.

Reply:
I've just spotted that the number of logins excluding logins immediately after a confirmation email bit of the report isn't working currently, and nobody's noticed...

Reply:
Right, it now looks like

```
Report up to 2021-01-12 15:00:00 +0000:
  - all registrations: 3
  - cookie consents:
    - False: 2
    - True: 1
  - feedback consents:
    - False: 2
    - True: 1
  - total number of logins: 81
  - accounts logged in to: 2
  - number of logins per account:
    - 2 times: 1
    - 79 times: 1
  - number of logins per account (excluding logins immediately after confirming email):
    - 2 times: 1
    - 77 times: 1

Report between 1970-01-01 and 2021-01-12 15:00:00 +0000:
  - all registrations: 3
  - cookie consents:
    - False: 2
    - True: 1
  - feedback consents:
    - False: 2
    - True: 1
  - total number of logins: 81
  - accounts logged in to: 2
  - number of logins per account:
    - 2 times: 1
    - 79 times: 1
  - number of logins per account (excluding logins immediately after confirming email):
    - 2 times: 1
    - 77 times: 1
```

Reply:
humm, yeah to be honest i've not really been looking at the report too much.
Perhaps like Erin says we should resolve this thread for the moment and get this PR merged.

Then start a separate ticket with a content person and try to overhaul the entire thing to be more readable?

Reply:
Or even ask if anyone actually looks at these numbers

Reply:
That format is _much_ improved!